.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "iojs"
\ No newline at end of file
+  - "5.1"
```

Comment:
Is it worth doing LTS (4.x) instead?

Certainly the LTS npm has some differences that have occasionally burned me on the starter projects.